tendrl/etcd_store.py

```diff
@@ -0,0 +1,19 @@
+import tendrl.utils as utils
+import etcd
+
+@utils.singleton
+class EtcdStore():
+    def __init__(self):
+        configs = utils.TendrlConfig()
+        host_name = configs.get("general", "etcd_server")
```

Comment:
If you are reading values from /etc/tendrl/tendrl.conf I feel the etcd server url is maintained as "etcd_connection"